lib/build.js

```diff
@@ -18,8 +18,8 @@ try {
   }
 }
 
-module.exports = ({ purgeFlag }) => {
-  const ngTwFile = path.normalize(path.resolve(process.cwd(), 'ng-tailwind.js'))
+module.exports = ({ purgeFlag, configPath }) => {
+  const ngTwFile = configPath ? path.normalize(path.resolve(process.cwd(),configPath) ): path.normalize(path.resolve(process.cwd(), 'ng-tailwind.js') ); 
```

Comment:
```
const ngTwFile = configPath
  ? path.normalize(path.resolve(process.cwd(), configPath))
  : path.normalize(path.resolve(process.cwd(), 'ng-tailwind.js'))
```

If you don't mind observing the style here and touching up the rest of the code accordingly, I would appreciate it. I follow standardJS, if that helps answer any questions. As you are the first contributor, I haven't integrated any eslint for this, sorry about that. I do appreciate the contribution, and I'm going to test out how it runs to see if I can merge and release this.

Reply:
Done.